spec/support/ansible_shared/automation_manager/refresh_configuartion_script_source.rb

```diff
@@ -14,27 +14,38 @@
   end
   let(:manager_class) { manager_class }
 
+  let(:tower_data) { Spec::Support::TowerDataHelper.tower_data }
+
+  let(:targeted_refresh_id) { tower_data['items']['targeted_refresh']['id'] }
```

Comment:
This PR extracts the actual data used in the specs. I tried to map it as well as possible to the data created by the Rake task, but still they are more or less independent. If I recall correctly, the ids used here were different from the _hello_repo_ and _another_repo_, so I created a separate entity for them in the YAML.

This can be resolved when merging with the other PR by using _hello_repo_ instead and changing the data in this spec. This will however require re-recording of the VCRs, which is what I didn’t want to do here. It’s the inconsistency between specs I was talking about with you earlier. :(